hazelcast/src/main/java/com/hazelcast/util/StringUtil.java

```diff
@@ -47,6 +47,7 @@
      * LOCALE_INTERNAL is the default locale for string operations and number formatting. Initialized to
      * {@code java.util.Locale.US} (US English).
      */
+    //TODO Use StringUtil.LOCALE_INTERNAL (language neutral) in Hazelcast 4
```

Comment:
Did you mean to say that we should use `java.util.Locale#ROOT`?

Reply:
Good catch :) Yes, it was the intention. I'll fix the comment.

Thanks for review.